src/sections/Projects/GetNighthawk/gnh.style.js

```diff
@@ -170,8 +170,13 @@ const Gnhwrapper = styled.section`
         .hero-section{
             padding: 3.125rem 0;
             .hero-text{
-                width: 50%;
-            }
+                width: 90%;
+		text-align:center;
```

Comment:
Do make it center-align from `767px` itself.

Reply:
Yeah Sure.